lib/tasks/grant_superadmin.rake

```diff
@@ -0,0 +1,14 @@
+namespace :superadmin do
+  desc 'Grant the superadmin role to specified users'
+  task 'grant', [:user_list] => [:environment] do |_cmd, args|
+    raise ArgumentError, 'A list of users is required: `rake superadmin:grant[user1,user2,...]`' unless args.user_list
+    args.to_a.each do |u|
+      user = User.find_by_user_key(u)
```

Comment:
If you rework this as

```ruby
user = User.find_by(email: u)
```

I believe you could remove the white list entry to .rubocop.yml above.